xml_inscriptions/bib_only/MA.Bos.MFA.G.19.318.xml

```diff
@@ -28,7 +28,36 @@
                      <p/>
                   </msItem>
                </msContents>
-               <physDesc><!--objectDesc, handDesc, decoDesc go here--></physDesc>
+               <physDesc>
+                  <objectDesc ana="#slab">
+                     <supportDesc ana="#stone.marble">
+                        <support><p>Marble relief slab of hero at a feast</p><dimensions type="surface" unit="cm">
+                           <height>13.5</height>
+                           <width>9</width>
+                           <depth/>
+                        </dimensions>
+                        </support>
+                        <condition ana="#fragment.single">
+                           <p><!--free text here, don't forget the ana attribute on the condition element--></p>
+                        </condition>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="1">
+                           <!--use a <locus> to narrow down part of inscription being discussed, i.e. campus-->
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc hands="1">
+                     <handNote ana="#impressed.inscribed"/>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote ana="unknownDec">
```

Comment:
We're moving away from "unknownoDec"; don't include anything here, if we don't have information/don't know what's on the stone.